src/composables/getDoctor.js

```diff
@@ -0,0 +1,12 @@
+import doctors from '../jsons/lekarze.json';
+
+const arr = doctors;
+
+const getDoctorsArray = () => {
+    const result = arr.map((element) => {
+        return element.Nazwa;
+    });
+    console.log(result);
```

Comment:
this array is in scope of function please write function which gets data from json and write to new array or sth like this